File: TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] num = {0,0};
        for(int i = 0;i<nums.length;i++){
            int chazhi = target-nums[i];
            for(int j = 0;j<nums.length;j++){
                if(chazhi==nums[j]&&i!=j){
                    num[0] = i;
                    num[1] = j;  
                    break;
                }
                    
            }
            if(num[0]!=0&&num[1]!=0){
                break;
            }
            
        }
        return num;
        
    }
}

File: Twosum.py
 class Solution:
      def twoSum(self, nums: List[int], target: int) -> List[int]:
           for n, num in enumerate(nums):
               if (target - num) in nums:
                     if nums.index(target - num) != n:
                         return [n, nums.index(target - num)]